src/api/config/telemetry-params.md

```diff
@@ -0,0 +1,47 @@
+# Telemetry Parameters
+
+Two Telemetries are supported: Substrate-based and File-based.
+
+## `telemetry.substrate`
+
+Enables Substrate active outbound telemetry.
+
+**Fields:**
+
+- **`name`:** The node's name to be seen on the telemetry
+  - **Type:** String
+  - **Required**
+- **`url`:** The url of the telemetry
```

Comment:
in other places you use "address", not "url". Shouldn't it be standartized?

Reply:
There is a difference between URL and address. Here it is exactly an URL. For example, for other `address`es, users might specify `localhost:8080` (host + port), but for URLs, users should specify the full path like `http://localhost:8081/telemetry` (protocol + host + port + path etc).

Reply:
ok, got it! thanks